tcf_website/views/search.py

```diff
@@ -135,9 +155,19 @@ def normalize_search_query(q: str) -> str:
         .filter(Q(number__isnull=True) | Q(number__regex=r"^\d{4}$"))
         # filters out classes that haven't been taught since Fall 2020
         .exclude(semester_last_taught_id__lt=48)
-        .order_by("-max_similarity")[:10]
+        .order_by("-max_similarity")
     )
 
+    # Apply filters
```

Comment:
Seems like time filters are not applied when you query something with a search, like searching "CS 2100", is that correct? If so, it should be fixed.